package.json

```diff
@@ -65,6 +65,7 @@
     "parcel-bundler": "^1.12.4",
     "prettier": "^2.2.1",
     "prettier-plugin-sh": "^0.6.0",
+    "shellcheck": "^1.0.0",
```

Comment:
How come we're adding this in?

Reply:
`shellcheck` is also offered as an npm package that downloads the latest release from GitHub - see https://github.com/gunar/shellcheck/blob/master/download.js - and I figured that this would make it easier for writing CI scripts, and also for contributors, since `shellcheck` would just automatically be pulled and live in `node_modules` instead of having to install it to the entire system.

Reply:
Ah gotcha. Sounds good to me!